source/css/sass/_mixins--media-query.scss

```diff
@@ -0,0 +1,15 @@
+@import "utility-functions";
+@import "variables/breakpoint";
+@import "variables/font";
+
+$mq-base-font-size: to-px($font-size-base-in-px);
+
+$mq-breakpoints: (
+  x-small: to-px($breakpoint-site-x_small),
+  small: to-px($breakpoint-site-small),
+  medium: to-px($breakpoint-site-medium),
+  wide: to-px($breakpoint-site-wide),
+  x-wide: to-px($breakpoint-site-x_wide),
+);
+
+@import "../../../node_modules/sass-mq/mq";
```

Comment:
Looked at Eyeglass, but couldn't see a way to make my IDE aware of those paths so seemed more of a hassle.

Reply:
Forgot to look at what the export does. Probably doesn’t work...

Reply:
This is similar to #26, which includes a (CSS) dependency.

Reply:
https://github.com/libero/pattern-library/pull/26/commits/abcab30e202ab17e69d8b9a1de60b7bb0c802534 should solve this.